modules/utxo-lib/src/bitgo/Unspent.ts

```diff
@@ -0,0 +1,106 @@
+import { TxOutput } from '..';
+import { Network } from '../networkTypes';
+import { toOutputScript } from '../address';
+import { UtxoTransactionBuilder } from './UtxoTransactionBuilder';
+
+/**
+ * Public unspent data in BitGo-specific representation.
+ */
+export interface Unspent {
+  /**
+   * Format: ${txid}:${vout}
+   */
+  id: string;
+  /**
+   * In network-specific representation
```

Comment:
This feels a bit terse - maybe "The network-specific encoded address"

Reply:
improved docstring